source/scripts/Background/index.ts

```diff
@@ -70,6 +71,7 @@ browser.runtime.onConnect.addListener(async (port: Runtime.Port) => {
     window.controller.utils.setFiat();
 
     port.onDisconnect.addListener(() => {
+      window.controller.wallet.setIsPopupOpen(false);
```

Comment:
Trying to set the popUp state here to false is always making a browser-polyfill error occur:

<img width="1324" alt="image" src="https://user-images.githubusercontent.com/21185537/220401227-fb4531ac-8d6f-4fe8-9ef4-eeaab64e5ee4.png">

This issue must be adequately addressed, without having an error display in the background each time it tries to change the state on port disconnection

Reply:
fixed